about-me.md

```diff
@@ -2,7 +2,7 @@
 
 ![Picture of me]()
 
-Hi, I'm Charlene Bernard. I'm a ________ at *__________*.
+Hi, I'm Charlene Trivedi. I'm a ________ at *__________*.
```

Comment:
I thought your name is Sheri? WHO ARE YOU!?!?!

Reply:
Charlene is my legal name. It's also what I'm called when I'm in trouble. By my mom.

Reply:
oh okay, IMPOSTER